BrowserKit/Sources/Common/Extensions/UIButton.ConfigurationExtension.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+extension UIButton.Configuration {
```

Comment:
Could you please rename the file `UIButton.Configuration+Extension` to match our other extensions?